src/test/CreateRecipe.test.js

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios'
+import React from 'react'
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import CreateRecipe from '../CreateRecipe'
+import { BrowserRouter } from 'react-router-dom'
+
+jest.mock('axios')
+
+describe('CreateRecipe', () => {
+    it('should create a recipe', async () => {
+        const mockedCreateRecipe = jest.fn(() => ({
+            status: 201,
+        }))
+        axios.post.mockImplementation(mockedCreateRecipe)
+
+        render(
+            <BrowserRouter>
+                <CreateRecipe />
+            </BrowserRouter>
+        )
+        await waitFor(() => screen.getByText('New recipe:'))
```

Comment:
```suggestion
        expect(await screen.findByText('New recipe:')).toBeInTheDocument()
```
use findBy to wait some element to appear in the DOM, it is designed for this and it will give more contextual info in case of error